### DISCUSSION TITLE
How to start nocodb after it crashed? [Apr 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1995] [Discord Threads]
> 🙋‍♂️ **XGeohound**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### can u create an issue in Github with some steps to reproduce?  
#  
> 🙋‍♂️ **XGeohound**  
#### Sure, the thing is that I do not want to pollute your repository with my question ^_^"    
#### nocodb crashed because of a config error on my side (giving the database user not all needed privileges). Now I fixed it and would just like to start the application/server again. Could not find any documentation on how to do that, so that's why I am here 🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/969334006426513428)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
